applications/launchpad_v2/src/components/Tag/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { useContext } from 'react'
+import { ThemeContext, CSSProperties } from 'styled-components'
+
+import Text from '../Text'
+
+import { TagProps } from './types'
+
+import { TagContainer, IconWrapper } from './styles'
+
+/**
+ * @name Tag
+ * @typedef TagProps
+ *
+ * @prop {ReactNode} [children] - text content to display
+ * @prop {CSSProperties} [style] - optional component styles
+ * @prop {'blue' | 'running' | 'warning' | 'expert'} [type] - tag types to determine color settings
+ * @prop {ReactNode} [icon] - optional SVG icon
+ * @prop {ReactNode} [subText] - optional additional tag text
+ *
+ * @example
+ * <Tag type='running' style={extraStyles} icon={<someIconComponent/>} subText='Mainnet'>
+ *    Running
+ * </Tag
+ */
+
+const Tag = ({
+  children,
+  style,
+  type,
+  variant = 'small',
+  icon,
+  subText,
+}: TagProps) => {
+  const theme = useContext(ThemeContext)
```

Comment:
please use `useTheme` helper (which you can import from styled-components)

Reply:
TS complains about this - properties not existing on `DefaultTheme`?

Reply:
Now fixed